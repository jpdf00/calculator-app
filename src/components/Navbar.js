import React from 'react';
import { Link } from 'react-router-dom';
import './style.css';

const Navbar = () => (
  <div className="Navbar">
    <h1>Math Magicians</h1>
    <div className="LinkGroup">
      <Link className="Link border" to="/">Home</Link>
      <Link className="Link border" to="/calculator">Calculator</Link>
      <Link className="Link" to="/quote">Quote</Link>
    </div>
  </div>
);

export default Navbar;
