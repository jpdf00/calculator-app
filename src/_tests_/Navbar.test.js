import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar Component', () => {
  test('It should render', () => {
    const navbar = renderer.create(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    ).toJSON();
    expect(navbar).toMatchSnapshot();
  });
});
