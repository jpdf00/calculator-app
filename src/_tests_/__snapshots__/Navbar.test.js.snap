// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component It should render 1`] = `
<div
  className="Navbar"
>
  <h1>
    Math Magicians
  </h1>
  <div
    className="LinkGroup"
  >
    <a
      className="Link border"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="Link border"
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="Link"
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</div>
`;
